word_usage.py

```diff
@@ -10,16 +10,46 @@
 max_frequency = 0
 words_only = False
 no_repeats = False
+split_apostrophe = False
+
+
+def clean(line, split=False):
+    # Remove apostrophes to split words cutted
+    line = line.replace("'", " ")
+
+    # Convert curly apostrophes to straight
+    line = line.replace(u"\u2018", "'")
+    line = line.replace(u"\u2019", "'")
+    line = line.replace(u"\u0060", "'")
+    line = line.replace(u"\u00b4", "'")
+
+    # Filter out symbols
+    line = re.sub("[^a-zA-Z\u00c0-\u024f\u1e00-\u1eff']", " ", line)
+
+    # Remove any double spaces introduced by last regex
+    line = re.sub("\ {2,}", " ", line)
+
+    line = line.lower()
+
+    if split:
+        return line.split()
+
+    return line
+
```

Comment:
`line` is a string so this function can return either a string or an array depending on whether the `split` parameter is set. But the code that calls this is expecting it to return an array.

Reply:
I did a second function to avoid that problem